src/checks/trigger-is-object.js

```diff
@@ -22,7 +22,9 @@ const triggerIsObject = {
 
     if (nonObjectResult !== undefined) {
       const repr = simpleTruncate(JSON.stringify(nonObjectResult), 50);
-      return [`Got a result missing that was not an object (${repr})`];
+      return [
+        `Got a non-object result in the array, expecting an object (${repr})`
```

Comment:
Since triggers are typically an array of objects (and we want to be consistent with tense), I'd change the end of this error to `expected only objects`)